### DISCUSSION TITLE
How to enable wei/pull to keep insiders fork in sync

### DISCUSSION THREAD

**Discussion by purpleclay** [squidfunk/mkdocs-material#4527] [Q&A]
I love mkdocs material insiders and have tried to follow the steps on syncing upstream with docker; see the [guide](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#with-docker). However, no matter what I do, the wei/pull app doesn't ever automatically sync upstream and create a pull request. I still have to manually do this from time to time to ensure I get the latest changes. I prefer the docker approach as it is much cleaner when developing locally.

I have also tried invoking wei/pull directly through the browser, with the URL: `https://pull.git.ci/process/${owner}/${repo}` by still no luck

Just wondering if anyone else is using this approach and has any tips. 🙏

**Comment by squidfunk** [comment]
I'd delegate this questions to other users of Insiders, as they have more experience syncing than me 😅
